### PR TITLE
fix(component): in page nav css

### PR DESCRIPTION
**Pull request by gepifanio**
PR to fix css using BEM and make it possible to override in FFAM

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 51b0c30969394cd47046b40c3c6a56dadc023f8d

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5cc65483ec440b0008326f7d

File: components/page/navigation/InPageNavigation.vue [hunk 1]
```diff
@@ -3,20 +3,20 @@
     :color="color"
     padded>
     <div>
```
Comment (TimMansell) [inline components/page/navigation/InPageNavigation.vue]: this `<div>` should have then block class `<div class="in-page-navigation">`

File: components/page/navigation/OutPageNavigation.vue [hunk 2]
```diff
@@ -3,20 +3,20 @@
     :color="color"
     padded>
     <div>
```
Comment (TimMansell) [inline components/page/navigation/OutPageNavigation.vue]: this `<div>` should have then block class `<div class="in-page-navigation">`

File: components/page/navigation/index.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
+.in-page-navigation {
+  display: flex;
```
Comment (TimMansell) [inline components/page/navigation/index.css]: Migrate styles to `&__container`

**Pull request merged**